logzio-logback-appender/pom.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.logz.appenders</groupId>
+        <artifactId>logzio-java-appenders</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>io.logz.logback</groupId>
+    <artifactId>logzio-logback-appender</artifactId>
+    <version>1.0.12-SNAPSHOT</version>
```

Comment:
It can't be 1.0.12-SNAPSHOT because 1.0.12 was already released. Let's use 1.0.13-SNAPSHOT